modules/virt-vm-wizard-fields-web.adoc

```diff
@@ -84,9 +84,16 @@ endif::[]
 
 |Flavor
 |Tiny, Small, Medium, Large, Custom
-|Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for *Flavor* are determined by the operating system.
+|An annotation in a virtual machine template with predefined values that determine the amount of CPU and memory allocated to the virtual machine, depending on the operating system associated with that template.
 
-.3+|Workload Type
+If you choose a custom template, you can override the `cpus` and `memsize` values in the template using custom values. The only way to use a custom flavor is to choose a custom template and change the predefined values to custom values.
```

Comment:
two nitpicks (still lgtm as it is, because it will not affect a user reading this doc),
a - it's not exactly "choose a custom template", it's choose the default template and customize only the cpus and memsize, (p.s. that will result in a custom template )
b - it's not exactly the only way, you can choose some flavor, and then after the VM is created set cpus and memsize in the details page (again this will result in a custom template)

Reply:
note:
I think this text will be more correct if we drop the "The only way to use a custom flavor is to choose a custom template and change the predefined values to custom values." because it's not the only way.

Reply:
I've made some corrections based on your suggestions. What do you think of those changes?

Additionally, do you think there's anything I need to change or add related to the Workload Type field? Or is the note I've added on performance issues enough?

Reply:
LGTM